app/src/main/java/com/imobile3/groovypayments/ui/dialog/PlaceholderDialogFragment.kt

```diff
@@ -0,0 +1,28 @@
+package com.imobile3.groovypayments.ui.dialog
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.content.DialogInterface
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import com.imobile3.groovypayments.R
+
+class PlaceholderDialogFragment : DialogFragment() {
```

Comment:
This class is not needed, there's already helper methods created in the code to help show a dialog

Reply:
Class has been deleted. I used the built in alertbox method built into the app.